spec-old-api/03-plugins/08-datadog/01-log_spec.lua

```diff
@@ -237,5 +237,6 @@ describe("Plugin: datadog (log)", function()
 
     local err_log = pl_file.read(helpers.test_conf.nginx_err_logs)
     assert.not_matches("attempt to index field 'api' (a nil value)", err_log, nil, true)
+    thread:join()
```

Comment:
this should be wrapped in a `finally` call, since currently it won't get executed if the assertion fails.